src/ui/React/ServerDropdown.tsx

```diff
@@ -35,24 +36,35 @@ export function ServerDropdown(props: IProps): React.ReactElement {
    * Checks if the server should be shown in the dropdown menu, based on the
    * 'serverType' property
    */
-  function isValidServer(s: BaseServer): boolean {
-    const purchased = (s instanceof Server && s.purchasedByPlayer) || s instanceof HacknetServer;
+  function isValidServer(baseServer: BaseServer): boolean {
+    /**
+     * isOwnedServer is true if baseServer is home, private servers, or hacknet servers. Note that home satisfies the
+     * condition: (baseServer instanceof Server && baseServer.purchasedByPlayer).
```

Comment:
I would specifically call out purchasedByPlayer, since the other part should be obvious.